src/model/randomizer.js

```diff
@@ -0,0 +1,13 @@
+import { getQuizQuestions } from '../api/getQuizQuestions.js';
+
+export async function getRandomQuizQuestions(animal, numberOfQuestions) {
+  const questionToRandomize = await getQuizQuestions(animal);
+  const randomQuestions = questionToRandomize.sort(() => Math.random() - 0.5);
+
+  if (numberOfQuestions < randomQuestions.length) {
+    const questionSet = randomQuestions.slice(0, numberOfQuestions);
+    return questionSet;
```

Comment:
To improve readability return randomQuestions.slice(0, numberOfQuestions);

Reply:
done.